packages/jupyter-ai-test/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@jupyter-ai/test",
+  "version": "2.18.1",
```

Comment:
The version of this test package should always match the version of `jupyter_ai`, since our version bumping script bumps all workspace packages (under the `packages/` directory) in lock-step. So it is fine to leave the version declared as `2.18.1` here.


Also, `_version.py` has been deleted as it should not be tracked in the source repo, per a separate review thread.